README.md

```diff
@@ -133,6 +133,52 @@ declare module '*.svg' {
 }
 ```
 
+## `nuxt-icon` component
+
+### Notes
+
+Due to how this component works, it only works on projects with vite (webpack is not supported)
+
+This component is very similar to `nuxt-icons` module's component, in fact the code was originally copied over from `nuxt-icons` module, but it has been modified to work in SSR.
```

Comment:
Let's add a link to the `nuxt-icons` repo and give credit to the author

Reply:
I think this was clear enough already that the credit goes to the nuxt-icons author ? just added the link